simuleval/evaluator/instance.py

```diff
@@ -46,6 +46,14 @@ def __init__(
         if self.dataloader is not None:
             self.source = self.dataloader[self.index]["source"]
             self.reference = self.dataloader[self.index]["target"]
+
+            if self.dataloader.tgt_lang_list is not None and self.index < len(
+                self.dataloader.tgt_lang_list
+            ):
+                self.tgt_lang = self.dataloader[self.index]["tgt_lang"]
+            else:
+                self.tgt_lang = None
```

Comment:
On second thought, this check here is redundant as we already do this check in the dataloader
```suggestion
            self.tgt_lang = self.dataloader[self.index]["tgt_lang"]
```